biorbd_optim/path_conditions.py

```diff
@@ -14,7 +14,7 @@ def regulation(var, nb_elements):
     @staticmethod
     def regulation_private(var, nb_elements, type):
         if len(var) != nb_elements:
-            raise RuntimeError(f"Invalid number of {type} " + str(len(var)) + " != " + str(nb_elements))
+            raise RuntimeError(f"Invalid number of {type} (" + str(len(var)) + "), the expected number as well " + str(nb_elements))
 
     def expand(self, other):
```

Comment:
"Expected number as well"? what do you mean? Did you intend to write "the expected size is"? :D